app/javascript/controllers/team_user_info_controller.js

```diff
@@ -0,0 +1,65 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["deActive", "active", "deActiveMenu"];
+
+  toggleDropdown() {
+    const menu = this.element.querySelector(".deactivate-menu");
+    menu.classList.toggle("hidden");
+  }
+  showConfirmation() {
+    if (this.deActiveMenuTarget.textContent.trim() === "Active") {
+      const confirmationModal = this.element.querySelector(
+        ".activate-confirmation"
+      );
+      confirmationModal.classList.remove("hidden");
+    } else {
+      const confirmationModal = this.element.querySelector(
+        ".deactivate-confirmation"
+      );
+      confirmationModal.classList.remove("hidden");
+    }
+
+    const menu = this.element.querySelector(".deactivate-menu");
+    menu.classList.add("hidden");
+  }
+
+  confirm(event) {
+    const buttonId = event.currentTarget.id;
+    if (buttonId === "de-activate-yes") {
+      console.log("deactivate");
+      const confirmationModal = this.element.querySelector(
+        ".deactivate-confirmation"
+      );
+      confirmationModal.classList.add("hidden");
+      this.deActiveTarget.classList.remove("hidden");
+      this.activeTarget.classList.add("hidden");
+      this.deActiveMenuTarget.textContent = "Active";
```

Comment:
Text content has to be Activate and Deactivate

Reply:
texts changed